### PR TITLE
Add duck type in README.md

### PR DESCRIPTION
**Pull request by sniperking1234**
Add duck type in README.md

### CODE REVIEW

File: apis/duck/README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Duck Types
 
 Knative leverages duck-typing to interact with resources inside of Kubernetes
-without explicit knowlage of the full resource shape. `knative/pkg` defines two
-duck types that are used throughout Knative: `Addressable` and `Source`.
+without explicit knowlage of the full resource shape. `knative/pkg` defines three
+duck types that are used throughout Knative: `Addressable`, `Binding`, and `Source`.
```
Comment (mattmoor) [inline apis/duck/README.md:5]: Format markdown:
```suggestion
without explicit knowlage of the full resource shape. `knative/pkg` defines
three duck types that are used throughout Knative: `Addressable`, `Binding`, and
`Source`.
```

File: apis/duck/README.md [hunk 2]
```diff
@@ -1,8 +1,9 @@
 # Duck Types
 
 Knative leverages duck-typing to interact with resources inside of Kubernetes
-without explicit knowlage of the full resource shape. `knative/pkg` defines two
-duck types that are used throughout Knative: `Addressable` and `Source`.
+without explicit knowlage of the full resource shape. `knative/pkg` defines
```
Comment (vaikas) [inline apis/duck/README.md:4]: nit: while here, would you mind:
s/knowlage/knowledge/

Comment (sniperking1234) [reply apis/duck/README.md]: Well-spotted!

**Pull request merged**